features/google/src/views/explore/explore.jsx

```diff
@@ -20,6 +40,7 @@ const ExploreView = () => {
         return (
             <Screen className="explore" layout="poly-standard-layout">
                 <List>
+                    <UnrecognizedCard />
```

Comment:
I'm not totally sure we will have some unrecognized data in the Google data. Let me check the Jira ticket for that.